tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -621,40 +622,11 @@ def validate_exposed_port_and_container_link(super_client, con, link_name,
     assert link_name == resp
 
 
-def check_round_robin_access(container_names, host, port):
-    con_hostname = container_names[:]
-    con_hostname_ordered = []
-
-    url = "http://" + host.ipAddresses()[0].address +\
-          ":" + port + "/name.html"
-
-    logger.info(url)
-
-    for n in range(0, len(con_hostname)):
-        r = requests.get(url)
-        response = r.text.strip("\n")
-        logger.info(response)
-        r.close()
-        assert response in con_hostname
-        con_hostname.remove(response)
-        con_hostname_ordered.append(response)
-
-    logger.info(con_hostname_ordered)
-
-    i = 0
-    for n in range(0, 10):
-        r = requests.get(url)
-        response = r.text.strip("\n")
-        r.close()
-        logger.info(response)
-        assert response == con_hostname_ordered[i]
-        i = i + 1
-        if i == len(con_hostname_ordered):
-            i = 0
-
-
 def validate_lb_service(super_client, client, env, services, lb_service, port,
-                        exclude_instance=None):
+                        target_services=None, hostheader=None, path=None):
+
+    if target_services is None:
```

Comment:
Since the validate_lb_service() is extended for supporting host_routing test cases and is also being used by the existing lb_services test cases , the optional target_services parameter which will not be passed by the lb_services test cases defaults to services  parameter in which case all the services linked to the LB are also targets for the URL that is to be validated.